steps/find-jira-issue/0.0.1/step.yml

```diff
@@ -0,0 +1,43 @@
+title: Find Jira Issue
+summary: |
+  Find the jira issue and add it as an environment variable
+description: |
+  Step that locates the a jira issue from the input and sets it as an environment variable for future steps
+website: https://github.com/jimmithy/bitrise-step-find-jira-issue
+source_code_url: https://github.com/jimmithy/bitrise-step-find-jira-issue
+support_url: https://github.com/jimmithy/bitrise-step-find-jira-issue/issues
+published_at: 2020-06-26T09:10:58.5386-04:00
+source:
+  git: https://github.com/jimmithy/bitrise-step-find-jira-issue.git
+  commit: 86ee7cdff9975d65bdee4b78ed2500773f1ee2f7
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- FIND_ISSUE_INPUT: $BITRISE_GIT_MESSAGE
```

Comment:
input name should use lower_snake_case and should not include word input

Reply:
changed to `find_issue_content`